modules/remove.js

```diff
@@ -0,0 +1,8 @@
+// removes first element having the condition
+export default function remove(collection, predicate) {
+  var index = collection.length;
+  while(index--){
+    if(predicate(collection[index])) collection.splice(index, 1)
```

Comment:
Following the conventions of other Underscore functions, it would be nice to also pass the index and the whole collection to the predicate.

```suggestion
    if (predicate(collection[index], index, collection)) collection.splice(index, 1);
```